Sources/OpenAPIRuntime/Errors/RuntimeError.swift

```diff
@@ -141,3 +141,43 @@ internal enum RuntimeError: Error, CustomStringConvertible, LocalizedError, Pret
 @_spi(Generated) public func throwUnexpectedResponseBody(expectedContent: String, body: any Sendable) throws -> Never {
     throw RuntimeError.unexpectedResponseBody(expectedContent: expectedContent, body: body)
 }
+
+/// HTTP Response status definition for ``RuntimeError``. 
+extension RuntimeError: HTTPResponseConvertible {
+    public var httpStatus: HTTPTypes.HTTPResponse.Status {
+        switch self {
+        case .invalidServerURL,
+                .invalidServerVariableValue:
+            .notFound
+        case .invalidExpectedContentType,
+                .missingCoderForCustomContentType,
+                .unexpectedContentTypeHeader:
+            .unsupportedMediaType
+        case .unexpectedAcceptHeader(_):
```

Comment:
Nit (probably will get picked up by the formatter): 
```suggestion
        case .unexpectedAcceptHeader:
```